tools/monorepo-utils/src/code-freeze/commands/get-version/index.ts

```diff
@@ -160,7 +160,7 @@ export const getVersionCommand = new Command( 'get-version' )
 			setOutput( 'monthlyMilestone', monthlyDevelopment.version );
 			setOutput(
 				'acceleratedReleaseDate',
-				acceleratedDevelopment.release.toISODate()
+				acceleratedRelease.release.toISODate()
```

Comment:
I think that the cause of the issue is that the object that should have been used here is `acceleratedRelease` instead of `acceleratedDevelopment`.